app/policies/implicit.json.erb

```diff
@@ -0,0 +1,13 @@
+{
+  "statement": [
+    {
+      "action": ["institution:read"],
+      "resource": ["institution/0"]
+    },
```

Comment:
That test existed because in order to be able to create an institution, you needed to access the index. Right now, a simple redirect is being done.

Please change the controller to avoid the forbidden and redirect the user to the institution creation screen. With that you don't need an implicit read access for a ghost institution.